airbyte-cdk/python/airbyte_cdk/models/airbyte_protocol.py

```diff
@@ -278,6 +278,7 @@ class Config:
         None,
         description="Additional and optional specification object to describe what an 'advanced' Auth flow would need to function.\n  - A connector should be able to fully function with the configuration as described by the ConnectorSpecification in a 'basic' mode.\n  - The 'advanced' mode provides easier UX for the user with UI improvements and automations. However, this requires further setup on the\n  server side by instance or workspace admins beforehand. The trade-off is that the user does not have to provide as many technical\n  inputs anymore and the auth process is faster and easier to complete.",
     )
+    protocol_version: Optional[str] = Field(None, description="the Airbyte Protocol version supported by the connector")
```

Comment:
Could you add a line to the CDK's CHANGELOG under the `Unreleased` header which says this was added? It'll get released on the next CDK change

Reply:
Updated, see [CHANGELOG.md](https://github.com/airbytehq/airbyte/pull/15340/files#diff-eaac011381105d313e921677aa92d6099b5e258b3b66dba8ee778d52b2127baf)